README.rst

```diff
@@ -1,107 +1,17 @@
-These are the functional tests for stf client-side. They check that OpenStack is running.
+Feature-verification-tests are made up of two parts: tests and callback plugins.
 
-The deployment templates are in infrared, and have the following names:
+The callback plugins output the test results in different ways.
+The custom_logger plugin takes a test-id from the task names and reports the results of the tests in a file, where each line contains a test ID and a result (pass/fail).
```

Comment:
after merging **https://github.com/infrawatch/feature-verification-tests/pull/298I** , we are no longer collect this files